packages/rocketchat-autotranslate/server/settings.js

```diff
@@ -1,4 +1,33 @@
 Meteor.startup(function() {
-	RocketChat.settings.add('AutoTranslate_Enabled', false, { type: 'boolean', group: 'Message', section: 'AutoTranslate', public: true });
-	RocketChat.settings.add('AutoTranslate_GoogleAPIKey', '', { type: 'string', group: 'Message', section: 'AutoTranslate', enableQuery: { _id: 'AutoTranslate_Enabled', value: true } });
+	RocketChat.settings.add('AutoTranslate_Enabled', false, {
+		type: 'boolean',
+		group: 'Message',
+		section: 'AutoTranslate',
+		public: true
+	});
+	RocketChat.settings.add('AutoTranslate_ServiceProvider', 'google-translate', {
+		type: 'select',
+		group: 'Message',
+		section: 'AutoTranslate',
+		values: [{
+			key: 'google-translate',
+			i18nLabel: 'AutoTranslate_Google'
+		}, {
+			key: 'deepl-translate',
+			i18nLabel: 'AutoTranslate_DeepL'
+		}],
+		enableQuery: [{_id: 'AutoTranslate_Enabled', value: true}],
+		i18nLabel: 'AutoTranslate_ServiceProvider',
+		public: true
+	});
+	RocketChat.settings.add('AutoTranslate_APIKey', '', {
```

Comment:
Instead of adding a setting here, you can [rename](https://github.com/assistify/Rocket.Chat/blob/develop/packages/assistify-ai/server/migrations.js) the existing setting. If the setting does not exist, add it.

For the PR towards Rocket.Chat, this renaming should be part of a migration (add a new [migration file with the next available number](https://github.com/assistify/Rocket.Chat/tree/develop/server/startup/migrations)).